Task1/Bank.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1
+{
+    class Bank
+    {
+        public static void BankAction()
+        {
+            decimal minMonthPayment = 100;
+            decimal debt = 700;
+
+            while (debt > 0)
+            {
+                PaymentInfo(ref debt, Payment(minMonthPayment));
+            }
+        }
+
+        public static void PaymentInfo(ref decimal debt, decimal payment)
+        {
+            debt -= payment;
+
+            if (debt > 0)
+            {
+                Console.WriteLine($"Current debt is {debt}");
+            }
+            else if (debt == 0)
+            {
+                Console.WriteLine("You dont have debt");
+            }
+            else if (debt < 0)
```

Comment:
This is the last state of debt variable. So 'else if' is excessive.
Else only without condition is enough.